install.bat

```diff
@@ -559,7 +559,7 @@ local function get_msvc_env_setup_cmd()
 		if exists(vcvarsall) then
 			local vcvarsall_args = { x86 = "", x86_64 = " x64", arm64 = " x86_arm64" }
 			assert(vcvarsall_args[vars.UNAME_M], "vars.UNAME_M: only x86, x86_64 and arm64 are supported")
-			return ('call "%s"%s'):format(vcvarsall, vcvarsall_args[vars.UNAME_M])
+			return ('call "%s"%s > NUL'):format(vcvarsall, vcvarsall_args[vars.UNAME_M])

```

Comment:
There are three other `call`s below (lines 586, 595, 604). Do we know that ` > NUL`s are not necessary in those, or do they need to be added as well?

Reply:
Probably the redirection is needed there too, but I have no way to test it. For my use-case, setting upLua and LuaRocks with a GitHub action, the fix in this PR is sufficient.